### PR TITLE
test/fix_pod_coverage_module_issue

### PR DESCRIPTION
**Pull request by ramin-fs**
https://trello.com/c/tnqpJGGO/358-test-fixpodcoveragemoduleissue

### CODE REVIEW

File: t/multiple_module_test.t [hunk 1]
```diff
@@ -0,0 +1,48 @@
+use strict;
```
Comment (michaelmueller-binary) [inline t/multiple_module_test.t:1]: :+1:

File: t/multiple_module_test.t [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+use Module::Path 'module_path';
+
+use Test::Pod::CoverageChange;
+use t::Nopod;
+use t::CorrectPod;
+use t::PodSyntaxError;
+use t::PartiallyCoveredPod;
+
+# Initializing variables
+my $test_module            = "t::Nopod";
```
Comment (chylli-deriv) [inline t/multiple_module_test.t]: I dont see any place that use these variables.

File: t/multiple_module_test.t [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "  at $main_module_path line 131."
+    );
+
+    Test::Pod::CoverageChange::pod_coverage_syntax_ok($test_module_path, {'t::Nopod' => 3, 't::PartiallyCoveredPod' => 1});
+    test_test("Handles multiple modules at once");
+    done_testing;
+};
+
+subtest 'modules order does not matter' => sub {
+    my $tests_modules_paths = ['t/PartiallyCoveredPod.pm', 't/CorrectPod.pm', 't/Nopod.pm'];
```
Comment (chylli-deriv) [inline t/multiple_module_test.t]: why not use `$test_module_path` here ?

**Review by chylli-deriv — APPROVED** [review]

File: t/multiple_module_test.t (hunk 3 above)
Comment (ramin-fs) [reply t/multiple_module_test.t]: I wanted to change the order and also don't want to override the `$test_module_path`.

File: t/multiple_module_test.t [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    Test::Pod::CoverageChange::pod_coverage_syntax_ok($tests_modules_paths, {'t::Nopod' => 3, 't::PartiallyCoveredPod' => 2});
```
Comment (chylli-deriv) [inline t/multiple_module_test.t]:
```suggestion
    Test::Pod::CoverageChange::pod_coverage_syntax_ok([$test_module_path->@[2,1,3] ], {'t::Nopod' => 3, 't::PartiallyCoveredPod' => 2});
```

**Review by chylli-deriv — APPROVED** [review]

**Review by qiangliu-deriv — APPROVED** [review]

**Pull request merged**